examples/3d/parenting.rs

```diff
@@ -11,18 +12,20 @@ fn main() {
         .run();
 }
 
-/// this component indicates what entities should rotate
+/// This component indicates what entities should rotate.
 #[derive(Component)]
 struct Rotator;
 
-/// rotates the parent, which will result in the child also rotating
+/// Rotates the parent, which will result in the child also rotating.
 fn rotator_system(time: Res<Time>, mut query: Query<&mut Transform, With<Rotator>>) {
     for mut transform in query.iter_mut() {
         transform.rotation *= Quat::from_rotation_x(3.0 * time.delta_seconds());
     }
 }
 
-/// set up a simple scene with a "parent" cube and a "child" cube
+/// Set up a simple scene with two hierarchies:
+/// - a "parent" cube and a "child" cube
+/// - a "parent" empty (= invisible entity, just a transform) and a "child" capsule
```

Comment:
```suggestion
/// - an invisible "parent" that just has a `Transform` and a "child" capsule
```

Reply:
The rewording is clearer, but I'm not sure for the `Transform` bit: I wrote without capital T on purpose to mean "a transform, in the conceptual way". Otherwise the comment might be inexact as it also has a GlobalTransform, but you are better judge :)
Same for the word "empty" which is used in other engines (Blender, Unity,...), but I understand you might want to reserve this word for something else later 👍

Reply:
Ah, that's useful context. I don't think "empty entity" is the right framing for Bevy in general: it implies that entities serve as containers for components in a way that's much more true in OOP. They're more like... identifiers that group data?

How about?
> an invisible "parent" (that has a transform, but not a mesh) and a "child" capsule"